packages/salesforcedx-vscode-soql/src/telemetry/index.ts

```diff
@@ -1,38 +1,11 @@
 /*
- * Copyright (c) 2020, salesforce.com, inc.
+ * Copyright (c) 2021, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode/out/src';
-import { JsonMap } from '@salesforce/ts-types';
-import * as vscode from 'vscode';
 
-export const telemetryService = TelemetryService.getInstance();
-
-export async function startTelemetry(
-  context: vscode.ExtensionContext,
-  hrtime: [number, number]
-): Promise<void> {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const extensionPackage = require(context.asAbsolutePath('./package.json'));
```

Comment:
Code moved from `index.ts` to `telemetry.ts` and this path calculation is modified to a relative path so that it can work in a webpack context.